### DISCUSSION TITLE
Data Update Tools Techniques

### DISCUSSION THREAD

**Discussion by scottc-WellSky** [microsoft/regorus#299] [Q&A]
My scenario isn't complex, but I'd like to check out existing options before I start creating my own solution. Are there tools like [OPAL](https://docs.opal.ac/) or [OPA's bundles w/data support ](https://www.openpolicyagent.org/ecosystem/opa-bundles/)to help keep Regorus data updated? I do see Regorus' bundle support, but it seems to only support policy changes.

Thanks for the great tool!

**Comment by anakrish** [comment]
Currently, Regorus is focused more on the core evaluation engine and less on the policy service components. There is not yet an official solution for keeping data/policies updated.
